data/language/nl-NL.yml

```diff
@@ -413,19 +413,19 @@ strings:
   396: Dempen
   397: Muziek afspelen
   398: "{COLOUR RED}Failliet!"
-  399: "{STRINGID} not allowed to wait for a full load at {STRINGID}"
-  400: "{STRINGID} has slipped to a halt on an incline"
+  399: "{STRINGID} mag niet op volledige lading wachten bij {STRINGID}"
+  400: "{STRINGID} is vastgelopen op een helling"
   401: "{STRINGID} accepteert nu {STRINGID}"
   402: "{STRINGID} accepteert geen {STRINGID} meer"
   403: New Transport Company!{NEWLINE}{STRINGID} starts construction near {STRINGID}!
   404: "{STRINGID} is not able to land at {STRINGID} due to the airport being unsuitable"
-  405: Citizens Celebrate!{NEWLINE}First {STRINGID} arrive in {STRINGID}!
-  406: Citizens Celebrate!{NEWLINE}First {STRINGID} delivery arrives in {STRINGID}!
-  407: Workers Celebrate!{NEWLINE}First {STRINGID} delivered to {STRINGID}!
-  408: New Vehicle Invented -{NEWLINE}“{STRINGID}”!
-  409: "{STRINGID} is promoted from “{STRINGID}” to “{STRINGID}”!"
+  405: Bevolking viert feest!{NEWLINE}Eerste {STRINGID} komt aan in {STRINGID}!
+  406: Bevolking viert feest!{NEWLINE}Eerste {STRINGID} levering komt aan in {STRINGID}!
```

Comment:
Misschien is 'Bewoners vieren feest!' iets natuurlijker?

Reply:
Dat kan, het is niet iets wat direct 1:1 uit Engels te vertalen is 😅